pkg/dockerregistry/server/simplelookupicsp.go

```diff
@@ -17,15 +20,18 @@ import (
 // *Context AlternativeBlobSourceStrategy interface.
 type simpleLookupICSP struct {
 	icspClient operatorv1alpha1client.ImageContentSourcePolicyInterface
+	idmsClient cfgv1client.ImageDigestMirrorSetInterface
 }
 
 // NewSimpleLookupICSPStrategy returns a new entity of simpleLookupICSP using provided client
 // to obtain cluster wide ICSP configuration.
 func NewSimpleLookupICSPStrategy(
```

Comment:
Do you have a plan for when to rename this function (and the related types)?

Reply:
Updated to rename the field in this PR.